docs/examples/general/expressions/expr_examples.ipynb

```diff
@@ -4,26 +4,32 @@
    "cell_type": "markdown",
```

Comment:
<ul><li>Again, there is <code>define_graph</code>. This sentence needs to be updated</li><li><strong>Note</strong> didn't render as bold</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2480/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Ad 1. Done</p><p>Ad 2. It's ok in the source. Maybe some rendering issue? I'll make sure that it is ok in the final build when I have it.</p>